notebooks/1.0-adm-data-preparation.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
You're using Github Desktop, right? There's a button that says "Update from master". Use it. Deal with merge conflicts locally, then push again and there should be no more conflicts, so I'll accept the changes.